image.yaml

```diff
@@ -1,9 +1,11 @@
 schema_version: 1
-name: "jboss-dataservices/datagrid-online-services"
+name: "datagrid-7/datagrid-services"
```

Comment:
@goldmann May I ask you to review changes in `image.yaml`. The next steps are:

1. Change this branch to `datagrid-services-dev`
2. Rename repo to `datagrid-7`. May I ask you to do it?
3. Update the links in the docs
4. Hit the build button

Reply:
There is no OSBS section in this file.